scripts/sync-assets.sh

```
@@ -0,0 +1,34 @@
+}
+
+sync_theme_css() {
+  local src="node_modules/@lgtm-hq/turbo-themes/packages/css/dist/themes"
+  if [[ ! -d "$src" ]]; then
+    echo "[sync-assets] turbo-themes CSS not found at $src — skipping" >&2
+    return 1
+  fi
+  rm -rf public/packages/css/dist/themes
+  mkdir -p public/packages/css/dist
+  cp -r "$src" public/packages/css/dist/themes
+  echo "[sync-assets] Synced $(ls public/packages/css/dist/themes | wc -l | tr -d ' ') theme CSS files to public/"
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)